release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2992,8 +2992,8 @@ link:https://access.redhat.com/solutions/6962977[{product-title} 4.10.18 contain
 
 * Previously, the Ingress Operator had unnecessary logic to remove a finalizer on `LoadBalancer-type` services in previous versions of {product-title}. With this update, the Ingress Operator no longer includes this logic. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2082161[*BZ#2082161*])
 
-[id="ocp-4-10-18-upgrading"]
-==== Upgrading
+[id="ocp-4-10-18-updating"]
+==== Updating
 
 To upgrade an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
```

Comment:
Does the word "upgrade" in the text need to match the heading and xref? like:
```suggestion
To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
```
The guidelines say to use "update" for consistency, but if "upgrade" is the _more_ consistent option here within the RN text, then that sense to go with. I trust your judgement 😄

Reply:
http://file.rdu.redhat.com/opayne/RNs-4.10.21/release_notes/ocp-4-10-release-notes.html#ocp-4-10-4-updating